artifacts/k8s-templates/konnectivity-agent.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
```

Comment:
In the future, we should be able to move the contents of this file and `konnectivity-proxy.yaml` to https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/addons/konnectivity-agent/daemonset.yaml and https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/manifests/konnectivity-server.yaml respectively. For now, it should be fine to keep the files here though.

Reply:
Agree. Right now, those files here to simplify validation process